packages/builders/src/messages/embed/UnsafeEmbed.ts

```diff
@@ -164,6 +168,14 @@ export class UnsafeEmbed {
 		);
 	}
 
+	/**
+	 * The hex color of the current color of the embed
+	 */
+	public get hexColor() {
+		if (!this.data.color) return undefined;
+		return `#${this.data.color}` as const;
```

Comment:
huh? That's not how you turn a base-10 into hex...
```suggestion
		return typeof this.data.color === 'number' ? `#${this.data.color.toString(16).padStart(6, '0')}` : null;
```